.circleci/config.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           command: cargo test --tests
       - run:
           name: Test messaging example
-          working_directory: examples/messaging
+          working_directory: examples/ez-message
```

Comment:
🤔

Reply:
aww. @armaniferrante approve this real quick?